batch/run-job/run-job.sh

```diff
@@ -50,11 +61,14 @@ az batch task create \
     --job-id myjob \
     --json-file tasks.json
 
```

Comment:
Regarding the error with the tasks.json file, you will need to pass the absolute path of the json file.